junebug/__init__.py

```diff
@@ -2,6 +2,28 @@
 Junebug is a system for managing text messaging transports via a RESTful HTTP
 interface.
 '''
+import os
+
+r = os.environ.get('REACTOR')
+if r == "SELECT":
+    from twisted.internet import selectreactor
+    selectreactor.install()
+elif r == "POLL":
+    from twisted.internet import pollreactor
+    pollreactor.install()
+elif r == "KQUEUE":
+    from twisted.internet import kqreactor
+    kqreactor.install()
+elif r == "WFMO":
+    from twisted.internet import win32eventreactor
+    win32eventreactor.install()
+elif r == "IOCP":
+    from twisted.internet import iocpreactor
+    iocpreactor.install()
+elif r == "EPOLL":
+    from twisted.internet import epollreactor
+    epollreactor.install()
```

Comment:
Not wild about the giant `elif`, but at least it's simple. Perhaps we should put it in a function and call the function (just so it's a bit packaged up?